src/main/java/org/spongepowered/api/item/merchant/TradeOffer.java

```diff
@@ -179,13 +203,48 @@ static Builder builder() {
         Builder maxUses(int maxUses);
 
         /**
-         * Sets the trade offer to be generated to grant experience upon use.
+         * Sets the trade offer to be generated to grant experience to
+         * the customer upon use.
          *
          * @param experience Whether the offer will grant experience
          * @return This builder
          */
         Builder canGrantExperience(boolean experience);
 
+        /**
+         * Sets how much experience the trade offer will grant to the merchant
+         * upon use.
+         *
+         * @param experience The amount of experience.
+         * @return This builder
+         */
+        Builder merchantExperienceGranted(int experience);
+
+        /**
+         * Sets the rate at which the generated trade offer's price will grow
+         * when demand for an item increases. Every time a villager trades, this
+         * value is multiplied by the current price of the offer and the demand
+         * bonus.
+         *
+         * <p>If this value is 0, trade demand will have no effect on this
+         * offer's price.</p>
+         *
+         * @param priceGrowthMultiplier The offer's rate of price growth
+         * @return This builder
+         * @see <a href="https://minecraft.gamepedia.com/Trading#Economics">the
+         * Minecraft Wiki for more detail on how the price growth multiplier
+         * applies to trade offers</a>
+         */
+        Builder priceGrowthMultiplier(double priceGrowthMultiplier);
+
+        /**
+         * Sets the demand bonus for this trade offer.
+         *
+         * @param bonus The offer's demand bonus
+         * @return This builder
+         */
+        Builder demandBonus(int bonus);
```

Comment:
There doesn't seem to be a getter for this value on the final TradeOffer.

Reply:
Now there is.